packages/design-system/src/Checkbox/Checkbox.styles.tsx

```diff
@@ -0,0 +1,165 @@
+import styled, { css } from "styled-components";
+
+const Variables = css`
+  --checkbox-color-label: var(--ads-v2-color-fg);
+  --checkbox-color-border: var(--ads-v2-color-border);
+  --checkbox-color-background-selected: var(--ads-v2-color-bg-brand-secondary);
+  --checkbox-color-border-selected: var(--ads-v2-color-bg-brand-secondary);
+  --checkbox-color-background-checkmark: var(
+    --ads-v2-color-fg-on-brand-secondary
+  );
+`;
+
+const Checkbox = css`
+  input[type="checkbox"] {
+    opacity: 0;
+    position: absolute;
+    left: 0;
+  }
+
+  & {
+    cursor: pointer;
+    position: relative;
+    padding-left: var(--ads-v2-spaces-7);
+    font-family: var(--ads-v2-font-family);
+    color: var(--checkbox-color-label);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &::before {
+    content: "";
+    position: absolute;
+    left: 0;
+    top: var(--ads-v2-spaces-1);
+    width: var(--ads-v2-spaces-5);
+    height: var(--ads-v2-spaces-5);
+    border: 1px solid var(--ads-v2-color-border);
+    border-radius: var(--ads-v2-border-radius);
+    box-sizing: border-box;
+  }
+
+  &::after {
+    content: "";
+    position: absolute;
+    left: 0;
+    top: var(--ads-v2-spaces-1);
+    width: var(--ads-v2-spaces-5);
```

Comment:
Let us not use spacing tokens for height and width - our spacing scale is designed to account for gaps between components, not to contain them.